### PR TITLE
[QSP] Add Simics QSP board support guide

### PR DESCRIPTION
**Pull request by LeanSheng**
Officially add Simics QSP board (X58Ich10Board) support.

Signed-off-by: LeanSheng <lean.sheng.tan@intel.com>

### CODE REVIEW

File: source/supported-hardware/qsp.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+|https://downloadcenter.intel.com/download/30403/Intel-Simics-Public-Release-Preview/|
+
+.. |https://downloadcenter.intel.com/download/30403/Intel-Simics-Public-Release-Preview/| raw:: html
+
+   <a href="https://downloadcenter.intel.com/download/30403/Intel-Simics-Public-Release-Preview/" target="_blank">https://downloadcenter.intel.com/download/30403/Intel-Simics-Public-Release-Preview/</a>
+
+
+.. note::
+  Currently QSP Simics supports Linux environment runtime only
```
Comment (rprangar) [inline source/supported-hardware/qsp.rst:34]: Currently SIMICS is supported on Linux environment only.

File: source/supported-hardware/qsp.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
+Please refer to the **Installation and Getting Started Guide.pdf** at bottom of the Intel Simics page to install the Intel Simics and get started!
+
+
+
+Boot to Shell on QSP Simics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Before trying on Slim Bootloader, please make sure that you have gone through all the step in **Installation and Getting Started Guide.pdf** to get yourself familiaze with the Simics!
```
Comment (rprangar) [inline source/supported-hardware/qsp.rst:46]: familiarize

File: source/supported-hardware/qsp.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Once it is done, proceed to load **SlimBootloader.bin** using Simics CLI method (in Simics terminal)::
```
Comment (rprangar) [inline source/supported-hardware/qsp.rst:48]: Expand this a bit. Something like

Launch SIMICS. 
SIMICS can be launched using CLI and the Slim Bootloader image can be specified as the bios image as follow.

File: source/supported-hardware/qsp.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+  simics> run-command-file targets/qsp-x86/firststeps.simics bios_image = "/home/(SlimBootloader.bin path)"
+
+Now test the newly built |SPN| image in QEMU emulator from command line::
+
+  qemu-system-x86_64 -machine q35 -nographic -serial mon:stdio -pflash Outputs/qemu/SlimBootloader.bin
```
Comment (rprangar) [inline source/supported-hardware/qsp.rst:54]: Why QEMU reference here?

**Review by rprangar — CHANGES_REQUESTED** [review]
Some comments.

File: source/supported-hardware/qsp.rst [hunk 5]
```diff
@@ -0,0 +1,70 @@
+.. _getting-started_qsp:
+
+
+QSP Simics Platform
+-----------------------
+
```
Comment (rprangar) [inline source/supported-hardware/qsp.rst:6]: Please add a line about the HW platform - x85 + ICH10

Comment (mauricema) [reply source/supported-hardware/qsp.rst:6]: I guess it is x58.

File: source/supported-hardware/qsp.rst [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+Now test the newly built |SPN| image in QEMU emulator from command line::
+
+  qemu-system-x86_64 -machine q35 -nographic -serial mon:stdio -pflash Outputs/qemu/SlimBootloader.bin
+
+Console outputs::
+
+    ============= Intel Slim Bootloader STAGE1A =============
+    ...
+    ============= Intel Slim Bootloader STAGE1B =============
+    ...
```
Comment (mauricema) [inline source/supported-hardware/qsp.rst:61]: keep the log to display platform ID and platform name

Comment (LeanSheng) [reply source/supported-hardware/qsp.rst:61]: sure, will fix this too

File: source/supported-hardware/qsp.rst [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+.. note:: QSP simics platform does not require stitching the |SPN| into an IFWI.
+
+
+
+Setting up QSP Simics
+^^^^^^^^^^^^^^^^^^^^^
+
+Please head over to this link to setup Intel Simics® Simulator Public Release Preview:
+
+|https://downloadcenter.intel.com/download/30403/Intel-Simics-Public-Release-Preview/|
```
Comment (rprangar) [inline source/supported-hardware/qsp.rst:26]: Use this link - https://software.intel.com/content/www/us/en/develop/articles/simics-simulator.html

File: source/supported-hardware/qsp.rst (hunk 5 above)
Comment (rprangar) [reply source/supported-hardware/qsp.rst:6]: yes, thanks.

**Pull request closed without merging**